ci/pipelines/apps-test-ecr.yml

```diff
@@ -217,6 +234,9 @@ groups:
   - name: toolbox
     jobs:
       - toolbox-image-to-test-ecr
```

Comment:
Should we rename this job for consistency too? `push-toolbox-to-test-ecr`?

Reply:
I was thinking that - but it does something slightly different. But yeah, on second thoughts I think you're right

Reply:
Cool 👍 We can bring the other apps in line with the naming scheme when we get to them.